### DISCUSSION TITLE
00921074-7740-5C09-7111-3E149E221D59

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#102] [General]
# Cha 110913−773444

Cha 110913−773444 is either a sub-brown dwarf or a rogue planet about 529 light-years in the constellation Chamaeleon. Like OTS 44, it is surrounded by a protoplanetary disk about 6.0 AU across

https://celestia.mobi/addon?item=00921074-7740-5C09-7111-3E149E221D59

**Comment by hiimdrram** [comment]
Good
